src/main/java/org/fenixedu/academic/service/services/bolonhaManager/CreateCompetenceCourse.java

```diff
@@ -42,8 +42,8 @@
 
     @Atomic
     public static CompetenceCourse run(String name, String nameEn, String acronym, Boolean basic, RegimeType regimeType,
-            CompetenceCourseLevel competenceCourseLevel, CompetenceCourseType type, String unitID, ExecutionSemester startSemester)
-            throws FenixServiceException {
+            CompetenceCourseLevel competenceCourseLevel, CompetenceCourseType type, String unitID,
+            ExecutionSemester startSemester, String code) throws FenixServiceException {
```

Comment:
This is an API change, the old signature must be kept (deprecated possibly) along with the new one.